### PR TITLE
Fix Svelte 3 HMR and Babel/runtime issues

### PR DESCRIPTION
**Pull request by DeMoorJasper**
## What kind of change does this PR introduce?

Fixes https://github.com/codesandbox/sandpack/issues/41

## Checklist

- [x] Documentation
- [x] Testing 
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

File: packages/common/src/utils/is-babel-7.ts [hunk 1]
```diff
@@ -36,13 +36,7 @@ export function isBabel7(
 
   if (dependencies.svelte || devDependencies.svelte) {
     const ver = dependencies.svelte || devDependencies.svelte;
-    const [maj] = ver.split('.');
-
-    if (maj) {
-      return +maj > 2;
```
Comment (DeMoorJasper) [inline packages/common/src/utils/is-babel-7.ts:42]: This doesn;t work with `^3.x.x`

File: packages/common/package.json [hunk 2]
```diff
@@ -72,7 +72,7 @@
     "react-scrollbars-custom": "^4.0.20",
     "react-spring": "^8.0.25",
     "react-textarea-autosize": "^6.1.0",
-    "semver": "^5.6.0",
```
Comment (DeMoorJasper) [inline packages/common/package.json:75]: old version didn't properly handle ranges

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5684/commits/afd825015027a2c7156bdf9b62281fd0866fe18c is at https://pr5684.build.csb.dev/s/new.

**Review by CompuIves — APPROVED** [review]
Ah, I see! Very nice find!

**Pull request merged**